src/utils/utils.ts

```diff
@@ -137,17 +134,41 @@ export const handleItemDescription = async (options: {
 
   // parent folder description
   if (filename === `${folderName}${DESCRIPTION_EXTENTION}`) {
+    console.log(filename)
     await updateParentDescription({ parentId, content });
   }
   // links description
-  else if (filename.endsWith(`.url${DESCRIPTION_EXTENTION}`)) {
+  else if (filename.endsWith(`${LINK_EXTENSION}${DESCRIPTION_EXTENTION}`)) {
+    const name = filename.slice(0, -(`${LINK_EXTENSION}${DESCRIPTION_EXTENTION}`.length))
     const item = items.find(({ name: thisName }) => name === thisName);
-    item.description = content;
+    if (item) {
+      item.description = content;
+
+    } else {
+      console.error(`Cannot find item with name ${name}`)
+    }
   }
-  // files description
+  // documents description
+  else if (filename.endsWith(`${GRAASP_DOCUMENT_EXTENSION}${DESCRIPTION_EXTENTION}`)) {
+    const name = filename.slice(0, -(`${GRAASP_DOCUMENT_EXTENSION}${DESCRIPTION_EXTENTION}`.length))
+    const item = items.find(({ name: thisName }) => name === thisName);
+    if (item) {
+      item.description = content;
+
+    } else {
+      console.error(`Cannot find item with name ${name}`)
+    }
+  }
+  // files and folders description
   else if (filename.endsWith(DESCRIPTION_EXTENTION)) {
-    const item = items.find(({ name: thisName }) => name === thisName.split('.')[0]);
-    item.description = content;
+    const name = filename.slice(0, -(DESCRIPTION_EXTENTION.length))
+    const item = items.find(({ name: thisName }) => name === thisName);
+    if (item) {
+      item.description = content;
+
+    } else {
+      console.error(`Cannot find item with name ${name}`)
+    }
```

Comment:
Can we factor this into a function to reduce repetition or is it better to keep the option of applying custom treatment for each case ?